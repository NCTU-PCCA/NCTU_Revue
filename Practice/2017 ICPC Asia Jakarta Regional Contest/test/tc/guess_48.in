68206493660 36
>><<<><<<<<<<<<>>><<><<><>>>><<<<<<=
