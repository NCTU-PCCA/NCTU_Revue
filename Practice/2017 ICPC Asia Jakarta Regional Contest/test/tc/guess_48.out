34103246830 51154870245 59680681953 55417776099 53286323172 52220596708 52753459940 52487028324 52353812516 52287204612 52253900660 52237248684 52228922696 52224759702 52222678205 52221637456 52222157830 52222418017 52222548111 52222483064 52222450540 52222466802 52222458671 52222454605 52222456638 52222455621 52222456129 52222456383 52222456510 52222456574 52222456542 52222456526 52222456518 52222456514 52222456512 52222456511
